pytorch_lightning/accelerators/ddp_backend.py

```diff
@@ -123,14 +139,116 @@ def __ddp_script_mode_setup(self):
 
         self.task_idx = 0
 
+        # wait for all the procs to start
+        sleep(2)
```

Comment:
Do the processes communicate on startup? I feel like a hardcoded sleep is not the optimal solution here